source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRColumnStackView.mm

```diff
@@ -0,0 +1,24 @@
+ //
+//  ACRColumnStackView
+//  ACRColumnStackView.mm
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import "ACRColumnStackView.h"
+
+@implementation ACRColumnStackView
+#if 1
+- (CGSize) intrinsicContentSize
+{
+    CGSize test;
+    if(self.relativeColumnWidth)
+        return test = CGSizeMake(self.relativeColumnWidth*3, 150);
+    else
+    test = super.intrinsicContentSize;
+    test.width = test.width * self.relativeColumnWidth;
+    return test;
+}
+#endif
+
```

Comment:
Is this test code? If so, it should be removed. If not, it should be cleaned up.